mp/src/vgui2/vgui_controls/CvarTextEntry.cpp

```diff
@@ -131,4 +149,5 @@ void CvarTextEntry::OnTextChanged()
     {
         PostActionSignal(new KeyValues("ControlModified"));
     }
+    ApplyChanges();
```

Comment:
I'd add a new line above this to help keep it visually cohesive. It'll be enforced in the new linter with requiring a newline after closing curly brackets.